FWCore/PluginManager/src/PluginManager.cc

```diff
@@ -47,7 +49,9 @@ namespace edmplugin {
         throw cms::Exception("PluginMangerCacheProblem")
             << "Unable to open the cache file '" << cacheFile.string() << "'. Please check permissions on file";
       }
+      pauseMaxMemoryPreload();
```

Comment:
Could we do this with some sort of sentinel? That way if an exception happens it gets turned back on.

Reply:
Done